pogobuf/pogobuf.client.js

```diff
@@ -804,7 +826,11 @@ function Client() {
 
     this.maxTries = 5;
     this.mapObjectsThrottlingEnabled = true;
-    this.mapObjectsMinDelay = DEFAULT_MAP_OBJECTS_DELAY * 1000;
+    this.mapObjectsMaxDelay = DEFAULT_MAP_OBJECTS_DELAY * 1000;
+    this.mapObjectsMinDistance = DEFAULT_MAP_OBJECTS_DISTANCE;
+    this.interactionThrottlingEnabled = true;
```

Comment:
Probably best to leave this to false to not break any applications (if backwards compatibility is a thing). If someone _really_ wants to have this kind of throttling enabled they should enable it themselves.